views/templates/panes/compiler.pug

```diff
@@ -22,6 +22,21 @@ mixin newPaneButton(classId, text, title, icon)
               .argmenuitem
                 span.argtitle Detailed Compiler Flags
                 span.argdescription Open a new window to edit verbose compiler flags
+    .btn-group.btn-group-sm.w-100(role="group" aria-label="Compiler revision picker")
+      noscript This feature requires JavaScript to be enabled.
```

Comment:
is this `noscript` strickly required? We don't have it anywhere else and I don't want to either have it in one place, or everywhere, unless it brings something to the table.

Reply:
Done (removed)